app/models/site_config.rb

```diff
@@ -11,6 +11,8 @@ class SiteConfig < RailsSettings::Base
   STACK_ICON = File.read(Rails.root.join("app/assets/images/stack.svg")).freeze
   LIGHTNING_ICON = File.read(Rails.root.join("app/assets/images/lightning.svg")).freeze
 
+  field :waiting_on_first_user, type: :boolean, default: !User.exists?
```

Comment:
The "default" will be set when the app is first spun up and it will be based on whether there are any users.